src/test/lib/MtuTest.cpp

```diff
@@ -0,0 +1,256 @@
+/*++
+
+    Copyright (c) Microsoft Corporation.
+    Licensed under the MIT License.
+
+Abstract:
+
+    MsQuic MTU Unittest
+
+--*/
+
+#include "precomp.h"
+#ifdef QUIC_CLOG
+#include "MtuTest.cpp.clog.h"
+#endif
+
+struct MtuTestContext {
+    MsQuicConnection* Connection {nullptr};
+
+    static QUIC_STATUS ConnCallback(_In_ MsQuicConnection* Conn, _In_opt_ void* Context, _Inout_ QUIC_CONNECTION_EVENT*) {
+        (static_cast<MtuTestContext*>(Context))->Connection = Conn;
+        return QUIC_STATUS_SUCCESS;
+    }
+};
+
+void
+QuicTestMtuSettings()
+{
+    {
+        //
+        // Test setting on library works
+        //
+        MsQuicSettings CurrentSettings;
+        uint32_t SettingsSize = sizeof(CurrentSettings);
+        TEST_QUIC_SUCCEEDED(
+            MsQuic->GetParam(
+                NULL,
+                QUIC_PARAM_LEVEL_GLOBAL,
+                QUIC_PARAM_GLOBAL_SETTINGS,
+                &SettingsSize,
+                &CurrentSettings));
```

Comment:
Maybe we add a `SetGlobal` and `GetGlobal` function to `MsQuicSettings` for these?